app/controllers/policies_controller.rb

```diff
@@ -9,6 +10,7 @@ def index
   end
 
   def edit
+    @category_id = @policy.plan.company_categories_id
```

Comment:
@GauravMakwana-BTC : You can directly use  this instead of creating a variable if it is used only at one place.